README.md

```diff
@@ -1,7 +1,40 @@
-# nulecule-library
-A set of [nulecule](https://github.com/projectatomic/nulecule)-ized applications that can be used for reference or as part of your applications
+# Nulecule library
 
-###Communication channels
+A set of [nulecule](https://github.com/projectatomic/nulecule)-ized applications that can be used for reference or as part of your deployment.
```

Comment:
I'm assuming that people coming to the library have gone through the `start guide` of Atomic App. We could add a definition of "Nuleculized" here if you wish!